languages/python/oso/tests/test_polar.py

```diff
@@ -845,6 +845,17 @@ class Post:
         next(results)
 
 
+def test_dot_path():
```

Comment:
It might be worth placing this in a separate test file called `test_partial.py` (like the sqlalchemy lib).

Rationale is it's a unit test, not a Polar integration test like everything else in here.